src/core/namespace.c

```diff
@@ -1073,6 +1074,22 @@ static int mount_sysfs(const MountEntry *m) {
         return 1;
 }
 
+static bool hidepid_value_supported(const char *opts) {
+        _cleanup_close_ int fd = -1;
+        const char *value;
+
+        if (!opts)
+                return true;
+
+        assert_se(value = startswith(opts, "hidepid="));
```

Comment:
`opts` may be "hidepid=default,subset=pid". Hence, spurious value may passed to `fsconfig()`.